lib/modules/apostrophe-assets/index.js

```diff
@@ -33,7 +33,7 @@
 // any unexpected effects. If this option is undefined, the APOS_MINIFY
 // environment variable is consulted.
 //
-// ### `serveStaticOptions`
+// ### `static`
 //
 // Pass options to the [serve-static](https://github.com/expressjs/serve-static/)
```

Comment:
Just reference `express.static` since we don't control what express actually does with that. Sorry for the nitpicking this is the last one (:

Reply:
Good point, changed with reference to https://expressjs.com/en/4x/api.html#express.static